src/shared/contexts/query/context.tsx

```diff
@@ -248,7 +260,7 @@ export const QueryProvider: FC = ({children}) => {
     const promise = fetch(url, {
       method: 'POST',
       headers,
-      body: JSON.stringify(body),
+      body: typeof body === 'string' ? body : JSON.stringify(body),
```

Comment:
When the body is already a string, no need to call `JSON.stringify()` again. Otherwise, `/api/v2private/query` will error